exporters/otlpjson/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id "java-library"
+    id "maven-publish"
+
+    id "ru.vyarus.animalsniffer"
+}
+
+description = 'OpenTelemetry Protocol JSON Exporters'
+ext.moduleName = "io.opentelemetry.exporter.otlpjson"
+
+dependencies {
+    compileOnly project(':sdk:trace')
```

Comment:
Would "provided" be a semantically more correct dependency declaration?

Reply:
I think provided is "expected to be there", e.g., tomcat, and optional is "add if you want". `compileOnly` roughly corresponds with optional.